tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/amqp/connect/AMQPMirrorConnectionTest.java

```diff
@@ -349,20 +512,41 @@ public void testProducerMessageIsMirroredWithoutCoreTunnelingUsesDefaultMessageF
       doTestProducerMessageIsMirroredWithCorrectMessageFormat(false);
    }
 
+   @Test
+   @Timeout(20)
+   public void testProducerMessageIsMirroredWithNoForwardAndTunneling() throws Exception {
+      doTestProducerMessageIsMirroredWithCorrectMessageFormat(true, true);
+   }
+
+   @Test
+   @Timeout(20)
+   public void testProducerMessageIsMirroredWithNoForwardAndTunelingAndWithoutTunneling() throws Exception {
```

Comment:
testProducerMessageIsMirroredWithNoForwardAndWithoutTunneling() ?

Reply:
Hehe, thanks for pointing this out :sweat_smile: